Schutzfile

```diff
@@ -2,7 +2,7 @@
   "fedora-35": {
     "dependencies": {
       "osbuild": {
-        "commit": "376cbffd136bc4ba86fc7c63697fa5b88fe3acef"
+        "commit": "cf8ebb46b29e7d3f3905342c8a5830a040339381"
```

Comment:
The commit does not have any RPMs associated with it, because CI was not run for it https://github.com/osbuild/osbuild/commit/cf8ebb46b29e7d3f3905342c8a5830a040339381 (note the `[skip ci]`).

Reply:
Thanks for the explanation! I have changed the commit hash.